lib/github_bitbucket_deployer/git.rb

```diff
@@ -90,12 +90,12 @@ def id_rsa_path
 
     def run(command)
       @logger.info "git run command: #{command}"
-      result = system("#{command} 2>&1")
+      result = `#{command} 2>&1`
       sleep 20
```

Comment:
Guessing its to give the system command time to have finished before continuing.